cluster-manifests/region2/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+- ../base/
```

Comment:
Any thought on doing something like:

```yaml
commonAnnotations:
  region: "2"
```
That would annotate everything deployed then with that, including the namespace without the need to have anything custom in the namespace.  The namespace would then no longer need to be a patch.

Reply:
Then you'd just have the per-region flux configuration (patch) in here.

Reply:
Done here  | Addressed from [2756c](https://github.com/mspnp/aks-secure-baseline/pull/183/commits/2756c97d271fb589d5170b82ae4fbb488588df01)

+ requested patched per region:

[84a9a](https://github.com/mspnp/aks-secure-baseline/pull/183/commits/84a9a58f674a7ef0d5e6a45fe124ca8c93990c59) and [d749b](https://github.com/mspnp/aks-secure-baseline/pull/183/commits/d749b9b42f00a8b7650d00d3531deba57530d1f9) +